Items Editing/BenK-msx'BuyOne'Phazma_Create pitch-rate (vari-speed) take envelope and render to new take.lua

```diff
@@ -0,0 +1,733 @@
+--[[
+ReaScript name: Create pitch-rate (vari-speed) take envelope and render to new take
+Author: BenK-msx'BuyOne'Phazma
```

Comment:
```suggestion
Author: BenK-msx, BuyOne, Phazma
```

And the filename should have one author, preferably who will be maintaining the package.